### PR TITLE
Declarative Shadow DOM: FOUC avoidance update

### PR DESCRIPTION
**Pull request by developit**
This is a small update to the Declarative Shadow DOM article that adds the recommended polyfill-compatible solution for preventing FOUC, courtesy of @mfreed7.

### CODE REVIEW

File: src/site/content/en/blog/declarative-shadow-dom/index.md [hunk 1]
```diff
@@ -373,6 +373,63 @@ Declarative Shadow Roots, it is only loaded and parsed once. The browser uses a
...
+```html
+<style>
+  x-foo:not(:defined) > * {
+    display: none;
+  }
+</style>
+```
+
+With the introduction of Declarative Shadow DOM, Custom Elements can be rendered
+or authored in HTML such that their shadow content in-place and ready before the
```
Comment (mfreed7) [inline src/site/content/en/blog/declarative-shadow-dom/index.md:395]: "shadow content in-place" --> "shadow content **is** in-place"

?

File: src/site/content/en/blog/declarative-shadow-dom/index.md [hunk 2]
```diff
@@ -373,6 +373,63 @@ Declarative Shadow Roots, it is only loaded and parsed once. The browser uses a
...
+shadow root's content from showing. However, removing that rule would cause
+browsers without Declarative Shadow DOM support to show incorrect or unstyled
+content until the Declarative Shadow DOM [polyfill](#polyfill) loads and
+converts the shadow root template into a real shadow root.
+
+Fortunately, this can be solved in CSS by modifying the FOUC style rule. In
+browsers that support Declarative Shadow DOM, the `<template shadowroot>`
+element is immediately converted into a shadow root, leaving no `<template>`
+element in the DOM tree. Browsers that don't support Declarative Shadow DOM
+preserve the `<template>` element, which we can use this to prevent FOUC:
```
Comment (mfreed7) [inline src/site/content/en/blog/declarative-shadow-dom/index.md:417]: "can use this to" --> "can use to"

**Comment by mfreed7** [comment]
Two small edits (to what I believe is my own text) but otherwise LGTM! Thanks!

**Comment by developit** [comment]
@googlebot I fixed it

(mason is a googler, email address is just mismatched)

**Review by jpmedley — APPROVED** [review]

**Pull request merged**